src/components/ui/PressCardList/Item/PressCardItem.tsx

```diff
@@ -0,0 +1,45 @@
+'use client'
+
+import Image from 'next/image'
+import { useFormattedDate } from '../../../../helpers/hooks/useFormattedDate'
+import { ArrowLink } from '../../ArrowLink/ArrowLink'
+import { IPressCard } from '@/helpers/types/pressItem'
+import styles from './PressCardItem.module.css'
+
+export const PressCardItem = ({
+  title,
+  outline,
+  publisher,
+  publishDate,
+  publishUrl,
+  imageUrl,
+}: IPressCard) => {
+  const { formattedDate } = useFormattedDate(publishDate, 'PressCard')
+
+  return (
+    <li className={styles.root}>
+      <div className={styles.header}>
+        <span className={styles.date}>{formattedDate}</span>
+        <h3 className={styles.title}>{title}</h3>
+      </div>
+      <div className={styles.body}>
+        <p className={styles.outline}>{outline.slice(0, 200) + '...'}</p>
+        <ArrowLink
+          href={publishUrl}
+          text={`Read on ${publisher}`}
+          className={styles.link}
+        />
+
+        <div className={styles.imgWrapper}>
+          <Image
+            width={720}
+            height={290}

```

Comment:
@Yura33-dev I may have caused some confusion regarding the `width` and `height` props of the next.js Image component earlier. To make it work here, we should use the `fill` prop instead. This will automatically apply absolute positioning and set the width and height of the image element to 100%.